### PR TITLE
feat(example): adds the extended pizza order example

### PR DESCRIPTION
**Pull request by rami-nk**
**Issue**
A more useful example to demonstrate how miranum connect works.

**Description**
This branch provides a new examples to the examples module. The example is a pizza order process with the integration of three different services, namely waiter, cook and email.

**Checklist**

* Code is easy to understand
* No obvious errors or bugs
* CI/CD Pipelines did run successfully

### CODE REVIEW

**Comment by dominikhorn93** [comment]
@rami-nk  Kannst du man den main branch in die branch hier mergen?

**Comment by rami-nk** [comment]
> @rami-nk Kannst du man den main branch in die branch hier mergen?

Done

**Review by Hafflgav — CHANGES_REQUESTED** [review]
So far I checked out the readme and process models. I think I have a couple of points worth discussing: 

There are a couple of things in the readme I would recommend to change. Besides I would refactor the processes a little. For instance, so that they have a parallel gateway for serving drinks and preparing the pizza. 

The bigger topic I wanted to discuss is the package structure. As I understood most of the domain logic is included in the "Core". I think it would be better to have separated projects for each of them because they are representing other domains which have different use cases from each other. 

To sum it up I have another open questions which is about the capabilities Miranum provides: 
Could the worker deploy on start the model to C7/C8? Then you do not need to write about this as a prerequisite. 

Lmk what you think about this and feel free to block some time in my calendar to chat about this. :) 

Another thing which just came to my mind: 
Imho we should move these examples to the Consulting Repo as soon as there is a Miranum-Connect dependency available through maven. I would also like to syndicate the readme description to our official documentation and add it as a guide over there. Wdyt?

File: examples/pizza-order/README.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+1. Start all services, namely the `WaiterC8Application`, `KitchenC8Application`, `EmailC8Application` and the `PizzaOrderFrontEnd`
+2. Make sure that you have a running instance of the Camunda8 Engine running. <br> *If not, just start the camunda-8 stack under `stack/camunda-8`.*
+3. Also make sure to deploy the `pizza-order-c8.bpmn` to the running Camunda8-Engine.
+4. Now navigate to http://localhost:5050 to view the Frontend-Application. You will end up seeing something like this:
+
+<br>
+<div align="center">
+    <img src="./../../images/pizza-order-frontend.png" alt="pizza-order-frontend" width="300" align="center">
+</div>
+<br>
```
[the remark this answers is not in the document]
Comment (rami-nk) [reply examples/pizza-order/README.md:23]: Resizing an image depends on the markdown engine used in the background (e.g. README.md on different website may look different). Using html in markdown is a common thing I would not start changing that at every place in the project. How would you otherwise change the size of an image?

File: examples/pizza-order/README.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+* `waiter`: responsible for placing the order, serving beverages and issuing the check
+* `kitchen`: responsible for the cooking
+* `email`: responsible for notifying the customer for the order confirmation
+* `frontend`: frontend to place order to the waiter service by REST
+
+## Run the project
+
+1. Start all services, namely the `WaiterC8Application`, `KitchenC8Application`, `EmailC8Application` and the `PizzaOrderFrontEnd`
+2. Make sure that you have a running instance of the Camunda8 Engine running. <br> *If not, just start the camunda-8 stack under `stack/camunda-8`.*
+3. Also make sure to deploy the `pizza-order-c8.bpmn` to the running Camunda8-Engine.
```
[the remark this answers is not in the document]
Comment (rami-nk) [reply examples/pizza-order/README.md:16]: Its not about miranum, you can tell the Camunda 7/8 client by a configuration-property to autodeploy your models

File: examples/pizza-order/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Hafflgav) [reply examples/pizza-order/README.md:23]: Imho the Markdown language is intended primarily to simplify writing without adding the complexity of HTML. Hence, I would not include HTML tags for sizing images. 

At Camunda it was considered as a bad practice and not allowed in the documentation. It can lead to more and more unwanted complexity in a readme file which should be rather simple. (e.g. by adding a .css file to make it look even better etc.) To me that is not the intent of such readme. 
But as said, that's a discussion point I wanted to raise here. 

I would simply use: 
`Description`
And make sure that the pictures itself are sized properly. (For instance: Rectangular diagrams around 500x1200px, and square diagrams around 500x500px)

**Review by Hafflgav — APPROVED** [review]
Looks good to me! Thanks for the good work 🚀
